### PR TITLE
Test_1 (Python Core)

### PR DESCRIPTION
**Pull request by baranovr**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    letter_count = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+            letter_count += 1
+
+    return letter_count
```
Comment (Bohdan-Salamakha) [inline app/main.py:22]: use count method
https://www.w3schools.com/python/ref_string_count.asp

**Review by Bohdan-Salamakha — CHANGES_REQUESTED** [review]
don't change tests!

File: app/main.py [hunk 2]
```diff
@@ -14,4 +14,7 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+
+    all_letters = phrase.lower().count(letter.lower())
+    return all_letters
```
Comment (Polyakiv-Andrey) [inline app/main.py:20]: return the answer immediately, do not create unnecessary variables

Comment (Polyakiv-Andrey) [inline app/main.py:20]: add an empty line at the end of the file

File: tests/test_main.py
Comment (Polyakiv-Andrey) [inline tests/test_main.py:1]: Screenshot 2023-07-20 at 16 05 00
restore this file to its original state

**Review by Polyakiv-Andrey — CHANGES_REQUESTED** [review]

**Pull request closed without merging**